### DISCUSSION TITLE
Add toggle to always show Injected Wallet

### DISCUSSION THREAD

**Discussion by cursedId0l** [rainbow-me/rainbowkit#1378] [Ideas]
### Feature Description
 I propose adding a toggle to always show the Injected Wallet option, regardless of whether other wallets are present or not.

### Why This Feature?
During the development process, many people utilize the impersonator extension. Currently, when other wallets are detected by the extension, the Injected Wallet option is automatically hidden. This forces us to disable all other wallets to make the Injected Wallet option visible again. As a result, we're faced with additional steps to switch between wallets, which introduces inefficiencies and slows down our overall development workflow.

Additionally, if users wanted to use the impersonator extension they will now be able to. 

By having a toggle that lets us decide if we want to always show the Injected Wallet, it will help streamline the development workflow, making Rainbow/Kit even more developer and user friendly.

### Implementation Suggestion
This feature could be added in the connectors config as so:

```typescript
const connectors = connectorsForWallets([
  {
    groupName: 'Suggested',
    wallets: [
      injectedWallet({ chains, alwaysShow: true }),
      metaMaskWallet({ projectId, chains }),
    ],
  },
]);
```

**Comment by DanielSinclair** [comment]
Hey! I understand the pain-point here, but unfortunately there is a deeper issue at play beyond RainbowKit. Each wallet extension (including Impersonator) competes to override the `window.ethereum` provider, so you won't always experience the expected behavior of Impersonator coming out ahead and thus being available to RainbowKit and Wagmi. We try to use `window.ethereum.providers` where possible (not supported by all wallets, including MetaMask) to meet the expected UX, but alas, it is recommended to only have 1 extension active at a time. Even if the `Browser Wallet` button was available, it likely wouldn't do what you'd expect. But I will add that Rainbow's extension (now in Beta) actually supports what you're looking for; watched wallets or hot wallets can both be connected to dApps. Shot you an invite!

**Reply by cursedId0l** [reply]
Thanks for the reply! I shared it with my colleagues. Where did you send the invite for rainbow's extension, I can't seem to find it?

**Reply by DanielSinclair** [reply]
@jtardioli Shot it over to the private relay email linked on your site. Feel free to email me at `daniel@rainbow.me` with another address if you didn't receive it. Subject line: "Your Beta invite is here 🌈"
